src/lib/node/message_handler.ts

```diff
@@ -31,6 +31,8 @@ export class NodeMessageHandler {
       case NodeCommand.abortFirmwareUpdate:
         await node.abortFirmwareUpdate();
         return {};
+      case NodeCommand.pollValue:
+        return await node.pollValue(message.valueId);
```

Comment:
The return value should always be wrapped in an object. `{ value: X }`. 

Also make sure that the result value can be JSON serialized.

Reply:
I was not sure if this actually returns the value as the docs are a bit unclear. I'll inspect the code to find out.